.github/workflows/test-js-packages.yml

```
@@ -183,10 +183,8 @@ jobs:
       - name: Set up test environment
         uses: ./.github/actions/setup
 
-      - name: Install playwright deps
-        run: |
-          npx playwright install
-          npx playwright install-deps
+      - name: Install Playwright
+        uses: ./.github/actions/install-playwright
```

```suggestion
        uses: ./.github/scripts/playwright-install.sh
```

Seems like the cache breaks something important here.